src/src/PluginStructs/P104_data_struct.cpp

```diff
@@ -517,28 +517,28 @@ bool P104_data_struct::saveSettings() {
 
-    zones[zoneIndex].size          = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_SIZE));
-    zones[zoneIndex].text          = enquoteString(webArg(getPluginCustomArgName(idx + P104_OFFSET_TEXT)));
-    zones[zoneIndex].content       = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_CONTENT));
-    zones[zoneIndex].alignment     = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_ALIGNMENT));
-    zones[zoneIndex].animationIn   = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_ANIM_IN));
-    zones[zoneIndex].speed         = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_SPEED));
-    zones[zoneIndex].animationOut  = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_ANIM_OUT));
-    zones[zoneIndex].pause         = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_PAUSE));
-    zones[zoneIndex].font          = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_FONT));
-    zones[zoneIndex].layout        = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_LAYOUT));
-    zones[zoneIndex].specialEffect = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_SPEC_EFFECT));
-    zones[zoneIndex].offset        = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_OFFSET));
-    zones[zoneIndex].brightness    = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_BRIGHTNESS));
-    zones[zoneIndex].repeatDelay   = getFormItemInt(getPluginCustomArgName(idx + P104_OFFSET_REPEATDELAY));
+    zones[zoneIndex].size          = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_SIZE));
+    zones[zoneIndex].text          = enquoteString(webArg(getPluginCustomArgName(index + P104_OFFSET_TEXT)));
+    zones[zoneIndex].content       = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_CONTENT));
+    zones[zoneIndex].alignment     = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_ALIGNMENT));
+    zones[zoneIndex].animationIn   = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_ANIM_IN));
+    zones[zoneIndex].speed         = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_SPEED));
+    zones[zoneIndex].animationOut  = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_ANIM_OUT));
+    zones[zoneIndex].pause         = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_PAUSE));
+    zones[zoneIndex].font          = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_FONT));
+    zones[zoneIndex].layout        = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_LAYOUT));
+    zones[zoneIndex].specialEffect = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_SPEC_EFFECT));
+    zones[zoneIndex].offset        = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_OFFSET));
+    zones[zoneIndex].brightness    = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_BRIGHTNESS));
+    zones[zoneIndex].repeatDelay   = getFormItemInt(getPluginCustomArgName(index + P104_OFFSET_REPEATDELAY));
```

Comment:
One tip;
You're calling a number of lines with a few nested function calls only differing slightly.

If you make a separate function only accepting 2 parameters (index and the offset) to get an int, the compiled binary will be smaller.
For this example it doesn't matter a lot, but I have seen it on the handling of the commands, that a small reduction in the calls there would matter quite a bit in the binary size.